playbooks/shellcheck.yaml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: all
+  tasks:
+    - name: Install requirements
+      become: yes
+      package:
+        name:
+          - golang-github-cpuguy83-go-md2man
+          - ninja-build
+          - meson
+          - ShellCheck
+          - bash-completion
```

Comment:
Nitpick: I think `bash-completion` isn't necessary for `ninja test`. We could potentially save a second or two on every CI run.